### PR TITLE
Error handling for getNewerPrefix in prepBetaRelease script

### PR DESCRIPTION
**Pull request by jadahiya-MSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

> Summarize the changes, including the goals and reasons for this change. Be sure to call out any technical or behavior changes that reviewers should be aware of.

> If this Pull Request should close/resolve any issues when merged, use [the special syntax for that](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) here.

### Main changes in the PR:

1.  Added in a throw error statement if `getNewerPrefix` completes the for loop without returning.

### CODE REVIEW

File: packages/teams-js/prepBetaRelease.js [hunk 1]
```diff
@@ -84,6 +84,10 @@ function getNewerPrefix(currBetaVer, currPkgJsonVer) {
       return currPkgPrefix;
     }
   }
+  //If we complete the loop and haven't returned, then their is an error in prefix comparisons
+  throw new Error(
+    `Error in compaison between Current Beta Version ${currBetaVer} with prefix ${currBetaPrefix} and Current Package Version ${currPkgJsonVer} with prefix ${currPkgPrefix}`,
```
Comment (TrevorJoelHarris) [inline packages/teams-js/prepBetaRelease.js:89]: should be 'comparison'

Comment (jadahiya-MSFT) [reply packages/teams-js/prepBetaRelease.js:89]: I knew I made a typo somewhere while I was typing this, good catch

**Review by AE-MS — APPROVED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

File: packages/teams-js/prepBetaRelease.js [hunk 2]
```diff
@@ -84,6 +84,10 @@ function getNewerPrefix(currBetaVer, currPkgJsonVer) {
       return currPkgPrefix;
     }
   }
+  //If we complete the loop and haven't returned, then their is an error in prefix comparisons
```
Comment (erinha) [inline packages/teams-js/prepBetaRelease.js:87]: nit: "their" -> "there"

**Review by AE-MS — APPROVED** [review]

**Review by TrevorJoelHarris — APPROVED** [review]

**Comment by jadahiya-MSFT** [comment]
Have to close this PR and reopen it in a new PR to trigger a proper build

**Pull request closed without merging**